benchmark/parsingcompetition.cpp

```diff
@@ -108,9 +108,14 @@ bool bench(const char *filename, bool verbose, bool just_data, int repeat_multip
        repeat, volume, !just_data);
   }
 
-  if (!just_data)
-    BEST_TIME("simdjson (dynamic mem) ", simdjson::dom::parser().parse(p).error(), simdjson::SUCCESS,
+  if (!just_data) {
+    auto inner=[](auto& str){
```

Comment:
If it's all right, could you name this something that matches its function? parse_dynamic() would be fine given the benchmark name.

It might be possible to do something like (dom::parser parser, parser.parse(str).error()) instead of inner(p), but I am less sure about that.

Reply:
Followed your first suggestion!